components/chat/messages.md

```diff
@@ -179,6 +243,145 @@ Control the width behavior of chat messages using the `MessageWidthMode` paramet
 * `MessageWidthMode.Standard` - Messages take up a portion of the available space for better readability (default behavior)
 * `MessageWidthMode.Full` - Messages span the full width of the chat container
 
+## Author and Receiver Message Settings
+
+The Chat component allows you to configure settings specifically for author messages (sent by the current user) and receiver messages (received from other users) using `ChatAuthorMessageSettings` and `ChatReceiverMessageSettings` components. These settings take precedence over global Chat settings, enabling different configurations for sent and received messages.
```

Comment:
"Lets" is preferable to "allows to" because "allows" carries the nuance of permission, while "lets" leans toward a capability or enablement.